Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -219,5 +219,5 @@ private func strokedRoundImage(fromImage image: UIImage?,
   circle.lineWidth = lineWidth
   circle.stroke()
 
-  return optionalize(UIGraphicsGetImageFromCurrentImageContext())?.imageWithRenderingMode(.AlwaysOriginal)
+  return UIGraphicsGetImageFromCurrentImageContext()?.imageWithRenderingMode(.AlwaysOriginal)
```

Comment:
arg, looks like a few migrations changes snuck into `Kickstarter.framework`. since im going to do that in a separate PR i think we can ignore any changes in `Kickstarter-iOS/` directory...